db/migrate/20210318203118_create_blogs.rb

```diff
@@ -0,0 +1,9 @@
+class CreateBlogs < ActiveRecord::Migration[6.1]
+  def change
+    create_table :blogs do |t|
+    t.string :title
+    t.text :content
+    t.timestamps null: false
```

Comment:
I'm not sure I understand why this schema doesn't match the one below in `db/schema.rb`?